docs/payment-gateways.rst

```diff
@@ -3,15 +3,16 @@
 Supported Payment Gateways
 ==========================
 
-Here is our list of supported payment gateways:
+Here is our list of supported payment gateways, and their configuration:
```

Comment:
Maybe ```and their configurations```?

Reply:
I think singular is correct, I'm not sure about the Oxford comma though.

Reply:
I think it's correct in this context, as we are talking about the configuration (singular) of the gateways. But I have a really little doubt. I will check on that.

I don't know what to say about the coma though, it sounds good like that :laughing:

Reply:
I just pushed some rewording, let me know what you think. The singular was correct after a check from a friend (UK English teacher :+1:), but the coma was unnecessary. Thanks :+1: 

https://github.com/mirumee/saleor/blob/1321ec282a638c05c4ad0c91791de9e3bcc4f5a9/docs/payment-gateways.rst#L6-L7

Reply:
Thanks. Looks great to me :p